libs/application/templates/driving-license/src/dataProviders/EligibilityProvider.ts

```diff
@@ -4,29 +4,36 @@ import {
   Application,
   SuccessfulDataProviderResult,
   FailedDataProviderResult,
+  formatText,
+  StaticText,
 } from '@island.is/application/core'
 import { m } from '../lib/messages'
+import { MessageDescriptor } from '@formatjs/intl'
 
 const extractReason = (
   requirements: { requirementMet: boolean; key: string }[],
 ): string => {
   return requirements
     .filter(({ requirementMet }) => !requirementMet)
     .map(({ key }) => key)
-    .map((key) => requirementKeyToMessage(key))
+    .map((key) => requirementKeyToMessageId(key))
     .join(', ')
 }
 
-const requirementKeyToMessage = (key: string) => {
+// TODO: we need a better way of getting the translated string in here, outside
+// of react. Possibly we should just make a more flexible results screen.
+// This string ends up being used as the paramejter displayed as the error message
+// for the failed dataprovider
```

Comment:
I didn't look where it was used but according to the description it's regarding the data provider reject reason, and it takes a StaticText argument as seen here https://github.com/island-is/island.is/blob/main/libs/application/templates/parental-leave/src/dataProviders/Children/Children.ts#L214-L216

Reply:
yes, however, i'm returning multiple messages / reasons for this check.

Reply:
So i'd need to have the display logic handle multiple reasons.

it would be nice to add that as a feature of the data provider UI - perhaps even style with bullet points. I could do that as a followup